--- radioactive-1.3.1/applet/Makefile.am.orig	Mon Jun 11 19:22:59 2001
+++ radioactive-1.3.1/applet/Makefile.am	Fri Feb  1 02:25:08 2002
@@ -1,4 +1,3 @@
-CXXFLAGS        = -g -Wall
 INCLUDES        = -I$(includedir) $(GNOMEMM_CFLAGS) $(GTKMM_CFLAGS) \
 			-I$(srcdir) -I$(top_srcdir) \
 			-DGNOMELOCALEDIR=\""$(datadir)/locale"\"
@@ -12,10 +11,10 @@
 radioactive_applet_LDADD	= $(top_builddir)/common/libracommon.a $(PANELMM_LIBS) $(GNOMEMM_LIBS)
 
 # Desktop entry
-Desktopdir		= $(datadir)/applets/Multimedia
-Desktop_in_files	= radioactive_applet.desktop.in
+AppletDesktopdir		= $(datadir)/applets/Multimedia
+AppletDesktop_in_files	= radioactive_applet.desktop.in
 if RA_APPLET
-Desktop_DATA		= $(Desktop_in_files:.desktop.in=.desktop)
+AppletDesktop_DATA	= $(AppletDesktop_in_files:.desktop.in=.desktop)
 @XML_I18N_MERGE_DESKTOP_RULE@
 endif
 
@@ -26,4 +25,4 @@
 gnorba_DATA		= $(gnorba_files)
 endif
 
-EXTRA_DIST		= $(Desktop_in_files) $(gnorba_files)
+EXTRA_DIST		= $(AppletDesktop_in_files) $(gnorba_files)
--- radioactive-1.3.1/standalone/Makefile.am~	Fri Feb  1 02:04:53 2002
+++ radioactive-1.3.1/standalone/Makefile.am	Fri Feb  1 02:04:53 2002
@@ -1,4 +1,3 @@
-CXXFLAGS        = -g -Wall
 INCLUDES        = -I$(includedir) $(GNOMEMM_CFLAGS) $(GTKMM_CFLAGS) \
 		-I$(srcdir) -I$(top_srcdir) \
 		-DGNOMELOCALEDIR=\""$(datadir)/locale"\"
--- radioactive-1.3.1/common/Makefile.am~	Fri Feb  1 02:05:23 2002
+++ radioactive-1.3.1/common/Makefile.am	Fri Feb  1 02:05:23 2002
@@ -1,4 +1,3 @@
-CXXFLAGS        = -g -Wall
 INCLUDES        = -I$(includedir) $(GNOMEMM_CFLAGS) $(GTKMM_CFLAGS) \
 			-I$(srcdir) -I$(top_srcdir) \
 			-DGNOMELOCALEDIR=\""$(datadir)/locale"\"
